tests/UI/campaigns/audit/BO/11_international.ts

```diff
@@ -0,0 +1,374 @@
+
+import {
+  boDashboardPage,
+  boLoginPage,
+  boLocalizationPage,
+  boLanguagesPage,
+  boLanguagesCreatePage,
+  boCurrenciesPage,
+  boCurrenciesCreatePage,
+  boCountriesPage,
+  boCountriesCreatePage,
+  boTranslationsPage,
+  type BrowserContext,
+  type Page,
+  utilsPlaywright,
+} from '@prestashop-core/ui-testing';
+
+const baseContext: string = 'audit_BO_international';
+
+describe('BO - International', async () => {
+  let browserContext: BrowserContext;
+  let page: Page;
+
+  before(async function () {
+    utilsPlaywright.setErrorsCaptured(true);
+
+    browserContext = await utilsPlaywright.createBrowserContext(this.browser);
+    page = await utilsPlaywright.newTab(browserContext);
+  });
+
+  after(async () => {
+    await utilsPlaywright.closeBrowserContext(browserContext);
+  });
+
+  it('should login in BO', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'loginBO', baseContext);
+
+    await boLoginPage.goTo(page, global.BO.URL);
+    await boLoginPage.successLogin(page, global.BO.EMAIL, global.BO.PASSWD);
+
+    const pageTitle = await boDashboardPage.getPageTitle(page);
+    expect(pageTitle).to.contains(boDashboardPage.pageTitle);
+
+    const jsErrors = utilsPlaywright.getJsErrors();
+    expect(jsErrors.length).to.equals(0);
+  });
+
+  it('should go to \'Localization\' page > Localization tab', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToLocalizationPage', baseContext);
+
+    await boDashboardPage.goToSubMenu(
+      page,
+      boDashboardPage.internationalParentLink,
+      boDashboardPage.localizationLink,
+    );
+    await boLocalizationPage.closeSfToolBar(page);
+
+    const pageTitle = await boLocalizationPage.getPageTitle(page);
+    expect(pageTitle).to.contains(boLocalizationPage.pageTitle);
+
+    const jsErrors = utilsPlaywright.getJsErrors();
+    expect(jsErrors.length).to.equals(0);
+  });
+
+  it('should go to \'Languages\' page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToLanguagesPage', baseContext);
+
+    await boLocalizationPage.goToSubTabLanguages(page);
+
+    const pageTitle = await boLanguagesPage.getPageTitle(page);
+    expect(pageTitle).to.contains(boLanguagesPage.pageTitle);
+
+    const jsErrors = utilsPlaywright.getJsErrors();
+    expect(jsErrors.length).to.equals(0);
+  });
+
+  it('should go to add new language page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToAddNewLanguages', baseContext);
+
+    await boLanguagesPage.goToAddNewLanguage(page);
+
+    const pageTitle = await boLanguagesCreatePage.getPageTitle(page);
+    expect(pageTitle).to.contains(boLanguagesCreatePage.pageTitle);
+
+    const jsErrors = utilsPlaywright.getJsErrors();
+    expect(jsErrors.length).to.equals(0);
+  });
+
+  it('should go to edit Language page', async function () {
```

Comment:
```suggestion```suggestion
  it('should go to \'Localization > Languages > Edit Language\' page', async function () {
```
```